common/src/primitives/compact.rs

```diff
@@ -20,6 +20,17 @@ use std::ops::Shl;
 #[derive(PartialEq, Eq, PartialOrd, Ord, Hash, Clone, Copy, Encode, Decode)]
 pub struct Compact(pub u32);
 
+impl Compact {
+    pub fn highest_value() -> Self {
+        Uint256([0xFFFFFFFFFFFFFFFF, 0xFFFFFFFFFFFFFFFF, 0xFFFFFFFFFFFFFFFF, 0xFFFFFFFFFFFFFFFF])
+            .into()
```

Comment:
Would `Uint256::MAX` work here?

Reply:
I originally used `MAX`, however since it's not a constant but rather a function, I thought an all-caps method felt wrong and so went with a lowercase function call. But then I didn't want to use `max()` since that might get confused with `std::cmp::max()`.

Reply:
> I originally used `MAX`, however since it's not a constant but rather a function, I thought an all-caps method felt wrong and so went with a lowercase function call. But then I didn't want to use `max()` since that might get confused with `std::cmp::max()`.

I meant use `Uint256::MAX`, which is already defined, in place of the rather unpleasing array constant.

Reply:
Thanks for that, I must have missed it. Fixed.